tools/CampaignConfigurationCheck/KeywordTrackingCode.mjs

```diff
@@ -0,0 +1,30 @@
+import chalk from 'chalk';
+
+export default class KeywordTrackingCode {
+	/**
+	 * @param {string} trackingCode
+	 */
+	constructor( trackingCode ) {
+		const testNumberRegexResult = trackingCode.match( /^\D*(\d{2})([-_])/ );
```

Comment:
Here you could also split at the `-` sign and examine the length and different parts with `^` and `$` regex chars. I always get a bit uneasy when I see a Regex without these "string boundaries" because it theoretically it could match anywhere in the string. Also, using boundaries makes the regex much faster because the regex engine does not have to try out all possible matches